src/plasmapy/diagnostics/langmuir.py

```diff
@@ -1393,7 +1394,7 @@ def get_EEDF(probe_characteristic, visualize: bool = False):
     probability = dIdV2[_filter] * np.sqrt(energy.to(u.eV).value)
 
     # Integration of the EEDF for the purpose of normalization.
-    integral = np.abs(np.trapz(probability, x=energy.to(u.eV).value))
+    integral = np.abs(trapezoid(probability, x=energy.to(u.eV).value))
```

Comment:
`trapz` was deprecated in NumPy v2.0.0, but `trapezoid` was only added in v2.0.0, so I imported it from `scipy.integrate` instead.